docs/modular-resources/configure.md

```diff
@@ -12,17 +12,17 @@ tags:
     "components",
     "services",
   ]
-description: "Add and configure a module from the Viam registry on your robot."
+description: "Add a modular resource to your robot by configuring it."
 no_list: true
 aliases:
   - "/program/extend/modular-resources/configure/"
 ---
 
-You can extend Viam by adding a module on your robot that provides one or more {{< glossary_tooltip term_id="resource" text="modular resources" >}} ([components](/components/) or [services](/services/)).
+To use a [modular resource](/modular-resources/) on your robot, you need to:
 
-You can [add a module from the Viam registry](#add-a-module-from-the-viam-registry), or you can [add a local module](#local-modules).
-
-See [Key Concepts of Modular Resource APIs](/modular-resources/key-concepts/) for more information.
+1. Configure a {{< glossary_tooltip term_id="module" text="module" >}}, either one [from the registry](#add-a-module-from-the-viam-registry) or a [local module](#local-modules).
+   This makes the modular resource available to the robot.
+1. Then configure the modular resource itself.
```

Comment:
You can add a modular resource directly without needing to do anything with the module that provides it. Most users will add the modular resource and not bother with any module-specific conf, in fact.

Conceptually - I agree - it makes sense to add the module, then add the modular resource it provides. But in practice, eng has designed the process so that the MRs can be added directly without needing awareness of its parent module.

To see in action:

App: Config > Create Component > `camera / realsense` > Add module > name it > Create > Poof module also added with default settings. No module conf required to start using realsense MR.

Reply:
Hmm. It is very streamlined but as you say you do click "Add module" before configuring the resource itself. That button configures the module. 

The current version of this intro section talks all about modules and says nothing about configuring the resource itself, so it sounds like we need to change that in some way, just need to figure out in which way, does that sound right?

Reply:
It does. Perhaps the heavy usage of 'Module' on this page should be 'modular resource' in more places, I could see that!

Reply:
Although, I guess I'd like to see the Add module button read "Add modular X" (component or service), since that's actually what is taking place (we came here from X tab, not Modules tab. We are adding to X tab, not modules tab -- although the providing module is also added to modules behind the scenes. Ex: We see a new camera component appear when we click, not the intel-realsense module appear).

The text itself "Add module" seems like a stretch to consider as a configuration step. Configuration seems to me to be more update management, and the upcoming [module environment variable support](https://github.com/viamrobotics/docs/pull/2102). Those steps configure the _module specifically_, so I think there is some cause to differentiate between configuring a module and its modular resource.

The real fix is better button text though.

Reply:
I thought "Add module" is correct because that button adds the module on the modules tab, and then you continue configuring the modular resource in place?

Reply:
I think you've really helped to clarify things with the latest edit. "Underlying module" is the language I've been looking for all this time, ha! Your callout about the module being added at the same time as the modular resource I think is also a great step forward, but have added a slight language suggestion as well. Thanks for thinking through this with me!